LOCALIZATION_CONTRIBUTORS

```diff
@@ -108,7 +108,7 @@ Rodrigo Rosado Rivial
 Sandra del Mar Soto Corderi
 Angel Gonzalez
 Siddhartha E Morales Guzman
-
+Jonathan Amador Lorence
 
```

Comment:
Carlos Alejandro Quiroz Solís

Reply:
@CarlosQbit Please open a new pull request if you'd like to join. Let me know if you need help with that. Thank you

Reply:
It's there something I have to do from my side. I am new contributing

Reply:
My name still is not on the list